.github/workflows/release.yaml

```
@@ -39,11 +39,11 @@ jobs:
         run: |
           wasm-bindgen --no-typescript --out-name bevy_game --out-dir wasm --target web target/wasm32-unknown-unknown/release/${{ env.binary }}.wasm
           cp -r assets wasm/
+
       - name: Package as a zip
-        uses: vimtor/action-zip@v1
-        with:
-          files: wasm
-          dest: ${{ env.binary }}.zip
+        working-directory: ./wasm
+        run: |
+          zip --recurse-paths ../${{ env.binary }}.zip .
```

```suggestion
        run: |
          zip --recurse-paths ${{ env.binary }}.zip wasm
```
Unless you feel strongly about that or there's a reason I'm missing, could you remove the `working-directory`? It really reduce readability for me.

---

If I don't, the zip archive will have a top level directory named "wasm" - which I find inelegant.